datafusion/ffi/README.md

```diff
@@ -0,0 +1,81 @@
+<!---
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing,
+  software distributed under the License is distributed on an
+  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  KIND, either express or implied.  See the License for the
+  specific language governing permissions and limitations
+  under the License.
+-->
+
+# `datafusion-ffi`: Apache DataFusion Foreign Function Interface
+
+This crate contains code to allow interoperability of Apache [DataFusion]
+with functions from other languages using a stable interface.
```

Comment:
Can we already define `stable` more precisely? Given frequent major releases of `datafusion`, a user may assume they can't load an FFI implementation compiled with `42` into a binary of version `43`.

OTOH, are there maybe subtle limitations regarding tokio runtime or something alike? (didn't research this topic TBH)

Reply:
You can run different versions! I've tested this in `datafusion-python` by backporting these changes into DF41 and I ran `datafusion-python` with DF42 and `delta-rs` with DF41. I'll work on the text.